### ISSUE TITLE
Fatally missing registry entries

### ISSUE THREAD

**Issue by Sciti** [Bletch1971/PixelmonInformation#13]
Game says "Fatally missing registry entries" when trying to connect via LAN. 
According to old issuies, this bug must be fixed in 0.2.8, looks like it's here again in 0.3.4

```
[18:52:00] [Client thread/INFO] [FML]: Injecting existing registry data into this client instance
[18:52:00] [Client thread/INFO] [FML]: Registry PokemonEntry: Found a missing id from the world jei:pixelmoninformation/pokemon_drops_flabebe_1
[18:52:00] [Client thread/ERROR] [FML]: Unidentified mapping from registry pixelmoninformation:pokemon_items
[18:52:00] [Client thread/ERROR] [FML]:     jei:pixelmoninformation/pokemon_drops_flabebe_1: 221
[18:52:00] [Netty Client IO #1/ERROR] [FML]: Network Disconnect: Fatally missing registry entries
[18:52:00] [Netty Client IO #1/FATAL] [FML]: Failed to connect to server: there are 1 missing registry items

```
This is the error(it's too smal for using pastebin i think)

Minecraft 1.12.2
Pixelmon-1.12.2-7.2.2
PixelmonInformation-1.12.2-7.2.0-0.3.4
jei_1.12.2-4.15.0.291

**Comment by Sciti** [comment]
Disabling JEI integration fixes it.. Waiting for new update with fixes.

**Comment by Bletch1971** [comment]
Thanks for the information, I will try to take a look and fix.

**Comment by Bletch1971** [comment]
@Sciti 
I am having problems trying to reproduce this issue.

How are you running this in minecraft
1. Single player
2. Dedicated server and one or more clients connecting to the dedicated server
3. Start a single player, then open to the LAN.

Also which version of forge are you running?

**Comment by Bletch1971** [comment]
@Sciti 
I have released a new version for pixelmon 7.2.0 and higher - 0.3.5.
Can you get the latest version and check if the issue you are having has been fixed?
